src/views/name.njk

```diff
@@ -56,13 +56,18 @@
           <p>
             You must check the name is not the same as an existing limited
             partnership on the
-            <a href="#/">Companies House register (link opens in a new tab)</a
+            <a href="https://find-and-update.company-information.service.gov.uk/company-name-availability">Companies House register (opens in a new tab)</a
             >. If it is, we will reject your application.
           </p>
+          <p>
+            If the name includes
+            <a href="https://www.gov.uk/government/publications/incorporation-and-names/annex-a-sensitive-words-and-expressions-or-words-that-could-imply-a-connection-with-government">sensitive words or expressions that require approval (opens in a new tab)</a
+            >. you must get permission before you register.
+          </p>
           <p>
             You should also use the
             <a href="https://www.gov.uk/search-for-trade mark" target="_blank"
-              >trade mark search (link opens in a new tab)</a
+              >trade mark search (opens in a new tab)</a
```

Comment:
The page in the Jira ticket is based on V5 of the prototype. Needs to use the 'insetText' nunjuck (https://github.com/companieshouse/limited-partnerships-prototype/blob/e10157ecbf2f52c1f6cc82bb3198c7693ea07c70/app/views/v5/registration/lp-name.html#L37)
Lines 49-77  can be replaced with this.

Reply:
Updated.